flink-core/src/main/java/org/apache/flink/configuration/ResourceManagerOptions.java

```diff
@@ -142,6 +142,13 @@ public class ResourceManagerOptions {
                                     + START_WORKER_MAX_FAILURE_RATE.key()
                                     + "') is reached.");
 
+    @Documentation.ExcludeFromDocumentation
```

Comment:
Why do you introduce this config option? Do you mean to allow users to configure it? If so, why do you exclude it from the documentation?

Reply:
It is inspired by the comments in the `ResourceManagerRuntimeServices` to make this configurable. But I think it do not have the requirements to change the default value now. But I can remove the annotation.

Reply:
```suggestion
    @Documentation.ExcludeFromDocumentation(
            "This is an expert option, that we do not want to expose in the documentation")
```

Reply:
thanks for your hint 👍🏻